src/components/staffing/ProjectCellText.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { MAP_PROJECT_NAMES } from '../../constants'
+
+export default class ProjectCellText extends React.Component {
```

Comment:
im assuming youve done a class component for consistency ?

Reply:
Yep